citation-matching/citation-matching-core-code/src/main/protobuf/citation_matching.proto

```diff
@@ -1,25 +0,0 @@
-//
-// (C) 2010-2012 ICM UW. All rights reserved.
-//
-package pl.edu.icm.coansys.citations;
-
-option java_package = "pl.edu.icm.coansys.citations.data";
-option java_outer_classname = "CitationMatchingProtos";
-
-message MatchableEntityData {
-  required string id = 1;
-  optional string author = 2;
-  optional string year = 3;
-  optional string pages = 4;
-  optional string source = 5;
-  optional string title = 6;
-  optional string rawText = 8;
-  optional string issue = 9;
-  optional string volume = 10;
-  repeated KeyValue auxiliary = 7;
-}
-
-message KeyValue {
-	optional string key = 1;
-	optional string value = 2;
-}
```

Comment:
I think that we should leave MatchableEntityData protocol buffer definition as it was.
When there will be needed some change in that class, this file will come in handy